lib/scrub/scrub_runner.rb

```diff
@@ -86,12 +98,29 @@ def run_file(file)
           Loader::HoldingLoader::Cleanup.new.on_success(:success, cleanup_data)
         end
       end
-    rescue
+    rescue => err
       # If the scrub failed, remove the file from local storage, that we may try again.
+      Services.logger.error err
       FileUtils.rm(downloaded_file)
       raise "Scrub failed, removing downloaded file #{downloaded_file}"
     end
 
+    # If we just scrubbed a umich mono we want to know what the line count was
+    # in the most recent umich mono so we can warn if that number is way +/-.
+    def count_lines_last_loaded(scrubber)
```

Comment:
Doesn't appear to be called anywhere except in the tests? I'd also advocate for moving this to `Scrub::AutoScrub` - it looks like all the data it's using is from `scrubber`

Reply:
Yeah I don't know why I did that. Not needed & removed.